openedx/core/djangoapps/content_tagging/api.py

```diff
@@ -38,7 +38,7 @@ def create_taxonomy(
 def set_taxonomy_orgs(
     taxonomy: Taxonomy,
     all_orgs=False,
-    orgs: list[Organization] = None,
+    orgs: list[Organization] | None = None,
     relationship: TaxonomyOrg.RelType = TaxonomyOrg.RelType.OWNER,
 ):
```

Comment:
Nit: missing return type
```suggestion
) -> None:
```